### ISSUE TITLE
How to decide the best yes/no distribution for each question

### ISSUE THREAD

**Issue by cjbayesian** [MarkDunne/33-questions#22]
Would it be best for each question to have approximately a 50/50 split of yes/no answers in the population? Presumably questions like "Do you live in a city?" would 'waste' distinguishing power since (as of 2010) more than half the global pop was urban. I think that you'd want all of the questions to be like "Are you female?", since close to 50% would be 1.

*\* EDIT: I'm a fool - didn't read the description closely enough. This was already addressed.

**Comment by seanmacentee** [comment]
yeah i'd agree... so things like this would work well to narrow date of birth:
were you born on an odd number day?
were you born on an odd number month?
were you born on an odd number year?
were you born in an odd number decade i.e. 30's 50's 70's etc...?

**Comment by thibaultcha** [comment]
@seanmacentee  :+1:  That's exactly what I was thinking in #11
